### PR TITLE
Obmin 353 redesign pop up of categories

### PR DESCRIPTION
**Pull request by Kyslytsia**
Redesign pop up of categories

### CODE REVIEW

File: src/components/category-button/CategoryButton.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { argTypes } from './arg-types';
+import { CategoryButton } from './index';
+import { IBtnCatygoryProps } from './types';
+
```
Comment (ASGladchenko) [inline src/components/category-button/CategoryButton.stories.tsx:7]: Category - may be?

File: src/components/category-button/arg-types.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { IArgTypes } from 'types';
+import { IBtnCatygoryProps } from './types';
+
```
Comment (ASGladchenko) [inline src/components/category-button/arg-types.ts:3]: IBtnCatygoryProps - IBtnCat**e**goryProps

File: src/components/category-button/arg-types.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+export const argTypes: IArgTypes<IBtnCatygoryProps> = {
+  top: {
```
Comment (ASGladchenko) [inline src/components/category-button/arg-types.ts:5]: same

File: src/components/category-button/index.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import * as Icon from '../icon';
+import * as Styles from './styles';
+import { IBtnCatygoryProps } from './types';
+
```
Comment (ASGladchenko) [inline src/components/category-button/index.tsx:4]: same

File: src/components/category-button/index.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+const CategoryButton = ({
+  top,
+  left,
+  text,
+  open,
+  setOpen,
+}: IBtnCatygoryProps) => {
+  return (
```
Comment (ASGladchenko) [inline src/components/category-button/index.tsx:12]: same

File: src/components/category-button/styles.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        bottom: -4px;
+        rotate: 180deg;
+      }
+      border: 2px solid ${theme.colors.categotyBtn.border};
+    `}
+
+    ${!theme.responsive.isDesktop &&
+    css`
+      display: none;
+    `}
```
Comment (ASGladchenko) [inline src/components/category-button/styles.ts:54]: Maybe it's better to do it in the component, then the element will not be on the page at all

File: src/components/icon/index.tsx [hunk 7]
```diff
@@ -4,13 +4,14 @@ export { ReactComponent as ArrowUp } from '../../assets/icons/arrow-up.svg';
 export { ReactComponent as Avatar } from '../../assets/icons/avatar.svg';
 export { ReactComponent as Bell } from '../../assets/icons/bell.svg';
 export { ReactComponent as Camera } from '../../assets/icons/camera.svg';
+export { ReactComponent as categoriesErrow } from '../../assets/icons/categories-errow.svg';
 export { ReactComponent as Check } from '../../assets/icons/check.svg';
```
Comment (ASGladchenko) [inline src/components/icon/index.tsx:8]: I think capitalized, and fix the grammar

File: src/config/theme/colors.ts [hunk 8]
```diff
@@ -149,6 +151,11 @@ const colorsDefault = Object.freeze({
   faux,
 });
 
+const categotyBtn = Object.freeze({
+  border: indigo,
```
Comment (ASGladchenko) [inline src/config/theme/colors.ts:155]: fix the grammar

File: src/config/theme/colors.ts [hunk 9]
```diff
@@ -162,6 +169,7 @@ const COLORS = Object.freeze({
   language,
   languageNew,
   modalColors,
+  categotyBtn,
   inboxMessage,
```
Comment (ASGladchenko) [inline src/config/theme/colors.ts:173]: fix the grammar

File: src/components/category-button/index.tsx [hunk 10]
```diff
@@ -0,0 +1,30 @@
+import * as Icon from '../icon';
+import * as Styles from './styles';
+import { IBtnCatygoryProps } from './types';
+
+const CategoryButton = ({
+  top,
+  left,
```
Comment (Wolshebnik) [inline src/components/category-button/index.tsx:7]: Are you sure what the top and left need?

**Review by Wolshebnik — COMMENTED** [review]
Look at the design and make the button change when you change devices
and do you remember, mobile-first?

**Pull request closed without merging**